doc/nrf/create_application.rst

```diff
@@ -0,0 +1,216 @@
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The process for creating an application in the |NCS| follows Zephyr's :ref:`Application Development principles <zephyr:application>` and the |NCS|'s :ref:`dev-model`.
+The following sections provide basic information based on these two sources.
+
+.. _create_application_structure:
+
+Application structure
+*********************
+
+Here are the files that most of the Zephyr and |NCS| applications include in its :file:`<app>` directory:
+
+.. code-block:: none
+
+   <app>
+   ├── CMakeLists.txt
+   ├── app.overlay
+   ├── prj.conf
+   └── src
+       └── main.c
+
+You can read more about each of these files in Zephyr's :ref:`Application Development overview <zephyr:application>`.
+
+.. _create_application_types:
+
+Application types
+*****************
+
+Based on where the :file:`<app>` directory is located, you will usually work with one of two types of applications in the |NCS|: workspace or freestanding.
+These types are adopted from :ref:`Zephyr application types <zephyr:zephyr-app-types>`.
+
+.. _create_application_types_workspace:
+
+Workspace application
+=====================
+
+A workspace application is located inside a west workspace, but outside of the SDK repository.
+This type of application specifies the |NCS| version through the :file:`west.yml` `west manifest file`_.
+This means that the |NCS| version setting specified in the workspace setting is not used to resolve the |NCS| installation.
+
+The workspace application has the following structure:
+
+.. code-block:: none
+
+   <home>/
+   └─── ncs/
+      ├── ...
+      └── applications/
+        └── <app>/
+          ├── CMakeLists.txt
+          ├── app.overlay
+          ├── prj.conf
+          └── src/
+            └── main.c
+
+You want to use this application type for the following development cases:
+
+* You want to take advantage of west to manage your own set of repositories.
+* You want to make changes to one or more of the repositories of the |NCS| when working on the application.
+
+For more information about workspace applications in the |NCS|, see the :ref:`development model page <dm_workflow_4>`.
+
+.. _create_application_types_freestanding:
+
+Freestanding application
+========================
+
+A freestanding application is located outside of a west workspace and is not using the `west manifest file`_ to specify the |NCS| version.
+Instead, it uses the |NCS| version specified in the workspace setting.
+This means that all freestanding applications in the same workspace will use the same |NCS| version.
+
+.. code-block:: none
+
+   <home>/
+   ├─── ncs/
+   │  └── ...
+   └─── <app>/
+      ├── CMakeLists.txt
+      ├── app.overlay
+      ├── prj.conf
+      └── src/
+        └── main.c
+
+You want to use this application type for the following development cases:
+
+* You are not planning to make any changes to any of the repositories of the |NCS| when working on the application.
```

Comment:
I think this is a pretty weak reason and doesn't make sense, I wouldn't personally recommend anyone uses a freestyle configuration, it's prone to breaking which is a big negative if you have multiple projects.

Reply:
What about:
`You prefer to use one copy of the |NCS| when working on one or more applications because of limited bandwidth.`